widget/embedded/src/test-utils/constants.ts

```diff
@@ -0,0 +1,132 @@
+export const TOKENS_COUNT = 20;
```

Comment:
Let's keep related pieces together. Move back this to `fixtures`. Constants doesn't make any sense in context of `test utils` and it's part of `fixture` actually.

Reply:
fixed